+nigeLab/@Block/init.m

```diff
@@ -70,11 +70,40 @@
 end
 blockObj.updateStatus('init');
 
+% Prior to link to data, check if a function handle for conversion has been
+% specified; if so, then do the conversion FIRST, then link it to data.
+if ~isempty(blockObj.MatFileWorkflow.ConvertFcn)
+   % Give the opportunity to cancel 
+   % (function handle can just be configured to [] once Block has been 
+   %  "converted" the first time)
+   printWarningLoop(3);
```

Comment:
Love this (printWarningLoop). Let's put it in +utils and change it so that it can also prompt a gui if necessary.

Reply:
Yep good point. I have been working on features/vidScore and of course forgot to swap back to this branch. So the updates are in that branch. Let me know if it is okay to go ahead and merge this one just to close it out; vidScore was branched from features/oldBlock_conversion anyways.